src/containers/pointsContainer.js

```diff
@@ -100,12 +99,13 @@ const PointsContainer = ({
     }
 
     if (isPinCodeOpen) {
-      dispatch(
-        openPinCodeModal({
-          navigateTo,
-          navigateParams,
-        }),
-      );
+      navigation.navigate({
+        routeName: ROUTES.SCREENS.PINCODE,
```

Comment:
Route name changed to name right?

Reply:
it is changed yes, but in most of our application we were using routeName, to keep the PR simple I am using the compat package provided by new navigation

Reply:
a lot of files in PR uses just this change, that helped keep using navigation the way we were using before. later, slowly we can migrate away from compat as well
![Screenshot 2022-09-08 at 4 28 41 PM](https://user-images.githubusercontent.com/6298342/189111181-793638f9-07bc-4ea7-883b-958ee3a3faa4.png)

Reply:
Understood 👍